packages/apps/contracts/TicTacToeApp.sol

```diff
@@ -1,10 +1,10 @@
 pragma solidity 0.4.25;
 pragma experimental "ABIEncoderV2";
 
-import "../lib/Transfer.sol";
+import "@counterfactual/contracts/contracts/libs/Transfer.sol";
 
 
-contract TicTacToe {
+contract TicTacToeApp {
```

Comment:
I think if we add a suffix, it should be `AppDefinition`

Reply:
Probably nicer if we don't require prefixes. In Solidity 0.5 we'll be able to enforce that the programmer inherits from some interface which we can call `interface AppDefinition { ... }` so it looks like `contract TicTacToe is AppDefinition { ... }`